### DISCUSSION TITLE
[ Section 2.3 - Instalação terraform] Criação de projeto

### DISCUSSION THREAD

**Discussion by cfilipee** [chgasparoto/curso-aws-com-terraform#114] [Q&A]
Boa noite,
Criei uma pasta para iniciar o projeto terraform, entretanto, recebo um erro dizendo que o projeto terraform não pode ser criado, pois não existe nenhum arquivo no local.

Quando o professor abre o editor dele, ja existem diversos arquivos criados, dos quais, não foram explicados como nem o porque.
Após pesquisar, consegui identificar que seria necessário criar o arquivo (backend.tf) para conseguir criar o projeto. De fato deu certo! Entretanto, gostaria de saber se esse é o jeito correto e, também, o porque do projeto do professor possuir os demais arquivos: git.ignore, README.md,  .terraform-version etc.

**Comment by chgasparoto** [comment]
Fala @cfilipee, consegue me esclarecer qual erro retornou e qual comando você estava tentando executar?

Sobre a utilização do arquivo `backend.tf` você só vai precisar desse arquivo se você setar um backend, e neste estágio do curso a gente não faz isso ainda.

Os outros arquivos que você mencionou são de ajuda/manutenção do repositório, não é relacionado ao Terraform em si.
